java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -66,10 +83,11 @@ public RdeImportUtils(
     this.escrowBucketName = escrowBucketName;
   }
 
-  private <T extends EppResource> void importEppResource(final T resource, final String type) {
+  public <T extends EppResource> ImmutableSet<Object> getIndexesForEppResource(T resource,
```

Comment:
This is typed with T extends EppResource, but it will do the wrong thing for DomainApplications, which do not have ForeignKeyIndexes.  Either check the type of T and throw an IllegalArgumentException if a domain application is passed, or use the more restrictive type of <T extends EppResource & ForeignKeyedEppResource> or similar.